README.md

```diff
@@ -1,31 +1,38 @@
-**Due to intel_pstate poor performances as of late, I have decided to set it to passive mode to make use of the acpi_cpufreq governors passthrough, keeping full support for turbo frequencies.**
+## Linux-TKG
```

Comment:
Nit: Usually, isn't it "linux-tkg"?

Reply:
Haha I don't know, I thought since it's a title x) Lemme change that

Reply:
You suggest lowercase "L" too ?

Reply:
Yeah. You're right that titles do use that casing, but the casing that the project is usually referred to with takes precedence.

Reply:
Done!